### PR TITLE
add tfe run container label documentation

### PR DESCRIPTION
**Pull request by p0pr0ck5**

### CODE REVIEW

File: content/source/docs/enterprise/admin/monitoring.html.md [hunk 1]
```diff
@@ -31,6 +31,15 @@ In addition to health-check monitoring, we recommend monitoring standard server
 - CPU
 - Disk
 
+As of the `v202108-1` release, TFE will attach additional labels to the Docker containers used to execute Terraform runs. These labels include:
+
+* `run_type`: The job’s run (either "plan" or "apply")
```
Comment (aaron-lane) [inline content/source/docs/enterprise/admin/monitoring.html.md:36]:
```suggestion
* `run_type`: The job’s run type (either "plan" or "apply")
```

File: content/source/docs/enterprise/admin/monitoring.html.md [hunk 2]
```diff
@@ -31,6 +31,15 @@ In addition to health-check monitoring, we recommend monitoring standard server
...
+* `run_id`: The job’s run ID (taking the form of "run-&lt;random string&gt;")
+* `workspace_name`: The workspace name under which the job is executing
+* `organization_name`: The organization under which the job is executing
```
Comment (aaron-lane) [inline content/source/docs/enterprise/admin/monitoring.html.md:39]:
```suggestion
* `workspace_name`: The name of the workspace under which the job is executing
* `organization_name`: The name of the organization under which the job is executing
```

**Review by aaron-lane — CHANGES_REQUESTED** [review]
A couple of tiny phrasing suggestions follow ✍🏼

**Comment by p0pr0ck5** [comment]
Thanks @aaron-lane, done! :+1:

**Review by aaron-lane — APPROVED** [review]

**Pull request merged**